es/resource_elasticsearch_index_test.go

```diff
@@ -80,6 +80,14 @@ resource "elasticsearch_index" "test" {
       output_unigrams  = false
     }
   })
+  analysis_char_filter = jsonencode({
+    my_char_filter_apostrophe = {
+      type     = "mapping"
+      mappings = [
+        "'=>"
```

Comment:
I think this just needs a trailing comma to pass the terraform formatting check. See the test here: https://github.com/phillbaker/terraform-provider-elasticsearch/actions/runs/3112373603/jobs/5053238514#step:10:82

Reply:
Thanks @phillbaker - I have committed the lint fix.